assets/texts/fr/fr.txt

```diff
@@ -0,0 +1,189 @@
+{
+  // manifest.json
+  "extensionName": {
+      "message": "Mastodon - Fédération simplifiée !",
+      "description": "Name of the extension.",
+      "hash": "8c716f7803f220414804500b514a1dc0"
+  },
+  "extensionNameShort": {
+      "message": "Mastodon - Fédération simplifiée !",
```

Comment:
The indentation now is a little broken. You have 6 spaces here, while two spaces on the line above.

Anyway, I get fix that myself, after merging.